clang/test/Driver/sycl-offload-with-split.c

```diff
@@ -122,13 +122,14 @@
 // CHK-UBACTIONS: 1: input, "[[INPUT]]", object, (host-sycl)
 // CHK-UBACTIONS: 2: clang-offload-unbundler, {1}, object, (host-sycl)
 // CHK-UBACTIONS: 3: linker, {0, 2}, image, (host-sycl)
-// CHK-UBACTIONS: 4: linker, {2}, ir, (device-sycl)
-// CHK-UBACTIONS: 5: sycl-post-link, {4}, tempfiletable, (device-sycl)
-// CHK-UBACTIONS: 6: file-table-tform, {5}, tempfilelist, (device-sycl)
-// CHK-UBACTIONS: 7: llvm-spirv, {6}, tempfilelist, (device-sycl)
-// CHK-UBACTIONS: 8: file-table-tform, {5, 7}, tempfiletable, (device-sycl)
-// CHK-UBACTIONS: 9: clang-offload-wrapper, {8}, object, (device-sycl)
-// CHK-UBACTIONS: 10: offload, "host-sycl (x86_64-unknown-linux-gnu)" {3}, "device-sycl (spir64-unknown-unknown)" {9}, image
+// CHK-UBACTIONS: 4: spirv-to-ir-wrapper, {2}, ir, (device-sycl)
```

Comment:
I'd suggest extending this file (or any other applicable LIT) with a `-###` check after archive unbundling so that the `spirv-to-ir-wrapper` command is tested in conjunction with `llvm-foreach`.
UPD: Another nuance here is that this would introduce an additional SPIR-V extension string test, which can make things harder to maintain. If this speculation is correct, I'd personally deem it OK to replace the exact list of extensions with a `{{.*}}` check.

Reply:
I've added `llvm-foreach` checks as requested.  As for `-llvm-spirv-opts` checking, there is currently no hook for adding these additional options within the toolchain.